frontend/src/app/blog/[name]/[id]/page.tsx

```
@@ -0,0 +1,121 @@
+              img_alt={section.image_alt_text}
+              text={section.text}
+              key={section.section_id}
+            />
+          ))}
+        </div>
+        <div className="px-16 hidden md:flex flex-col gap-4 sticky  h-fit top-7">
+          <a href={`#${blogData.intro.section_id}`} className="capitalize">
+            {blogData.intro.heading}
+          </a>
+          {blogData.sections.map((section) => (
+            <a href={`#${section.section_id}`} className="capitalize">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)